app/Http/Controllers/SearchController.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class Search extends Controller
+{
+    //
```

Comment:
I see you've created this Search controller, but it doesn't have any action methods. I'd rename this to `SearchController` and add a `public method displayResults` that we can route to.